gremlin-core/src/main/java/org/apache/tinkerpop/gremlin/jsr223/GremlinScriptChecker.java

```diff
@@ -228,11 +244,20 @@ public Optional<String> getRequestId() {
             return null == requestId ? Optional.empty() : Optional.of(requestId);
         }
 
+        /**
+         * Gets the value of the request identifier supplied using the {@code with()} source step. If there are
```

Comment:
nit: prefer `link` for javadoc when referring elsewhere in the codebase that we can hyperlink to and `code` when referring to the item being documented, external libs or jdk classes especially on public methods like this one.

Reply:
and 2 years ago you preferred to use `code` :) and I did the same

fixed in https://github.com/apache/tinkerpop/pull/1843/commits/143a2bf5b39a26412e501bc3a3f6f564d32d9afd

Reply:
ha - i code reviewed myself i guess